src/main/java/minicraft/screen/TitleDisplay.java

```diff
@@ -44,7 +44,6 @@ public TitleDisplay() {
 				else Game.setDisplay(new WorldGenDisplay());
 			}),
 			new SelectEntry("Options", () -> Game.setDisplay(new OptionsMainMenuDisplay())),
-            new SelectEntry("minicraft.display.skin", () -> Game.setDisplay(new SkinDisplay())),
```

Comment:
Is this required?

Reply:
I could use conditionals and re-adjust the title screen, it's just that there were too many options and I saw it as a simple solution

Reply:
Okay